chia/seeder/crawler.py

```diff
@@ -140,7 +140,7 @@ async def connect_task(self, peer: PeerRecord) -> None:
 
         async def peer_action(peer: WSChiaConnection) -> None:
             peer_info = peer.get_peer_info()
-            version = peer.get_version()
+            version = peer.get_chia_version_str()
```

Comment:
I would call get_version here (as mentioned earlier) and then use the standalone `chia_version_str_from_str`